web/src/recent_view_ui.js

```diff
@@ -1265,16 +1265,14 @@ export function initialize({on_click_participant, on_mark_pm_as_read, on_mark_to
         ui_util.convert_enter_to_click,
     );
 
+    // NOTE: Actual mechanism to muting / un-muting happens in click_handlers.js
+    // via user_topics.set_visibility_policy_for_element.
```

Comment:
I'd prefer to just have these duplicated, it's way easier to reason about if one doesn't rely on click handler propagation and it's like <20 lines of code we'd need to add to achieve it.